app/views/contexts/summary/group.py

```diff
@@ -95,10 +101,13 @@ def _build_blocks_and_links(
                 )
 
             elif block["type"] == "ListCollector":
-                section = schema.get_section(location.section_id)
+                section: Optional[ImmutableDict] = schema.get_section(
+                    location.section_id
+                )
 
+                summary_item: Optional[ImmutableDict]
                 if summary_item := schema.get_summary_item_for_list_for_section(
-                    section_id=section["id"],
+                    section_id=section["id"],  # type: ignore
```

Comment:
add a comment to explain why we added the `#type :ignore`

Reply:
Added 👍